src/configModel/configModel.vala

```diff
@@ -149,6 +149,12 @@ namespace SwayNotificatonCenter {
          */
         public int notification_window_width { get; set; default = 500; }
 
+        /** GtkLayerShell margins around the notification center */
+        public int control_center_margin_top { get; set; default = 0; }
+        public int control_center_margin_bottom { get; set; default = 0; }
+        public int control_center_margin_left { get; set; default = 0; }
+        public int control_center_margin_right { get; set; default = 0; }
```

Comment:
Should probably check if the value is below zero. It "works" when setting a negative value but I don't think that it's a desirable feature